storybookjs/storybook
# Fix Horizontal Scrollbar of DownPanel

Issue: #2387

## What I did
Extend flex property to "1 1 auto", which sets a flex-basis of auto instead of 0%. While this wont work in IE10 or below, those are not supported by storybook anyway :)

## How to test
* start locally
* with firefox (v57 and above), navigate to http://localhost:9010/?selectedKind=Addon%20Info.React%20Docgen&selectedStory=Comments%20from%20PropType%20declarations&full=0&down=1&left=1&panelRight=1&downPanel=storybook-addon-background%2Fbackground-panel
* try scrolling horizontally in the right panel

Is this testable with jest or storyshots?
no, since styles with css modules are not snapshot. 

Does this need a new example in the kitchen sink apps?
no, there is already one that shows the issue (see link above)

Does this need an update to the documentation?
no
